### PR TITLE
fix setParameters error

### PR DESCRIPTION
**Pull request by agrass**
Hi, 
This is a solution to setParameters error, it seems that that there are a lot of border cases when We set the parameters of the Camara, because depending the device it could change and raise a RuntimeError. I have an error in only one specific device Galaxy Tab A 7.0.

This fix will not change any logic, it continue working the same way. The only difference is that is catching RuntimeException at the moment of setting the parameters.
 
I was searching for the best solution, but adding try catch it was the only solution I found, because I can't replicate this in all the devices: https://stackoverflow.com/questions/39878068/exception-java-lang-runtimeexception-setparameters-failed

Let me know what do you think guys.

### CODE REVIEW

File: android/src/main/java/com/google/android/cameraview/Camera1.java [hunk 1]
```diff
@@ -275,7 +275,12 @@ void setPictureSize(Size size) {
         }
         if (mCameraParameters != null && mCamera != null) {
             mCameraParameters.setPictureSize(mPictureSize.getWidth(), mPictureSize.getHeight());
-            mCamera.setParameters(mCameraParameters);
+            try{
+              mCamera.setParameters(mCameraParameters);
+            }
+            catch(RuntimeException e ) {
+              Log.e("Camera1", "setParameters failed", e);
```
Comment (sibelius) [inline android/src/main/java/com/google/android/cameraview/Camera1.java]: better keep the same tag for this log

Log.e("RNCamera")

Comment (agrass) [reply android/src/main/java/com/google/android/cameraview/Camera1.java]: you are right, I'm going to change that.

Comment (agrass) [reply android/src/main/java/com/google/android/cameraview/Camera1.java]: Actually it use "CAMERA_1::", should I conserve that?

**Comment by agrass** [comment]
Changed log names, let me know if there are something else that I need to change.

**Pull request merged**